recipes/default.rb

```diff
@@ -123,7 +123,11 @@
     # Prevent duplicate resources
     # rubocop:disable Style/Next
     unless node['hadoop'][envfile]["#{svc}_log_dir"] == "/var/log/hadoop-#{log_dir}" || (
-           hdp22? && node['hadoop'][envfile]["#{svc}_log_dir"] == "/var/log/hadoop/#{log_dir}")
+           hdp22? && (
+             (node['platform_family'] == 'debian' && node['hadoop'][envfile]["#{svc}_log_dir"] == "/var/log/hadoop-#{log_dir}") ||
```

Comment:
i've stared at this code for like 15min trying to follow the conditionals.  I think this whole line is unecessary because the "dash" comparison here is redundant to the first clause of the ``unless`` where it is checked by itself.